new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -31,18 +33,34 @@
       .orElse(REPO_DUMMY_VALUE);
   public static final String REPO_EMAIL = Optional.ofNullable(System.getenv("REPO_EMAIL"))
       .orElse(REPO_DUMMY_VALUE);
+
+  // jenkins constants
   public static final String BUILD_ID = Optional.ofNullable(System.getenv("BUILD_ID"))
       .orElse("");
   public static final String BRANCH_NAME_FROM_JENKINS = Optional.ofNullable(System.getenv("BRANCH"))
       .orElse("");
+
   public static final String K8S_NODEPORT_HOST = Optional.ofNullable(System.getenv("K8S_NODEPORT_HOST"))
         .orElse(assertDoesNotThrow(() -> InetAddress.getLocalHost().getHostName()));
   public static final String GOOGLE_REPO_URL = "https://kubernetes-charts.storage.googleapis.com/";
   public static final String LOGS_DIR = System.getenv().getOrDefault("RESULT_ROOT",
       System.getProperty("java.io.tmpdir")) + "/diagnosticlogs";
   public static final String PV_ROOT = System.getenv().getOrDefault("PV_ROOT",
       System.getProperty("java.io.tmpdir") + "/ittestspvroot");
+
+  // NGINX constants
   public static final String NGINX_RELEASE_NAME = "nginx-release" + BUILD_ID;
   public static final String STABLE_REPO_NAME = "stable";
   public static final String NGINX_CHART_NAME = "nginx-ingress";
+
+  // MII image constants
+  public static final String MII_BASIC_WDT_MODEL_FILE = "model-basicdomain-singlecluster-sampleapp-wls.yaml";
+  public static final String MII_BASIC_IMAGE_NAME = REPO_USERNAME.equals(REPO_DUMMY_VALUE)
```

Comment:
The REPO_NAME will either be a value or empty, so please always add it.  We need this for the kind-based tests where there is no need for a REPO_USERNAME, but we do have a REPO_NAME.

Reply:
yes, I just noticed that...fixing and testing on kind jenkins job

Reply:
done